Emby.Server.Implementations/ScheduledTasks/Tasks/RefreshMediaLibraryTask.cs

```diff
@@ -58,6 +71,8 @@ public Task Execute(CancellationToken cancellationToken, IProgress<double> progr
 
             progress.Report(0);
 
+            _imageGenerator.GenerateSplashscreen(Path.Combine(_applicationPaths.DataPath, "splashscreen.webp"));
```

Comment:
Will this override a manually-uploaded splashscreen?

Reply:
No, the manually uploaded splashcreen gets saved as `splashscreen-upload.extension`. This path gets saved in branding.xml. A GET request to `/Branding/Splashscreen` checks if a file exists at this location. If not, splashscreen.webp is used. Generating a new splashscreen during the scheduled task will not override the path in branding.xml or the file at `splashscreen-upload.extension`